packages/app/public/oauth2-redirect.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<!-- 
+     This is copied from https://github.com/swagger-api/swagger-ui/blob/master/dist/oauth2-redirect.html
+     and is used by plugins/api-docs swagger-ui-react to implement the required redirect_uri for
+     OAuth2 Authorization Code flow.
+-->
```

Comment:
As this comes from swagger-api/swagger-ui which is licensed under Apache 2.0, I wonder if we should include the license header here?

Reply:
I don't see the need for that. The file didn't have any comment in it and I only added this per @freben's suggestion. It's still weird to have to copy something from upstream.

Reply:
changed this to a documentation-only update